app/src/model/accounts.js

```diff
@@ -0,0 +1,66 @@
+const connection = require('./connection');
+
+class UserEntity {
+    /**
+     *
+     * @param {number} id
+     * @param {string} username
+     * @param {string} password
+     */
+
+    constructor(id, username, password) {
+        this.id = id;
+        this.username = username;
+        this.password = password;
+    }
+}
+/**
+ * @param {PostEntity} post
```

Comment:
Brak dokumentacji jest lepszy niż dokumentacja która kłamie. A że my jesteśmy zajebiści to zrobimy dokumentację która nie kłamie ;)

Reply:
poprawione